### ISSUE TITLE
dark mode support

### ISSUE THREAD

**Issue by senadir** [WordPress/twentytwenty#25]
What the possibility of adding dark mode support to the theme?
with Gutenberg having partial [dark mode support](https://developer.wordpress.org/block-editor/developers/themes/theme-support/#dark-backgrounds), and with [`prefers-color-scheme`](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme) supported in most of the latest browsers, I think we can add support to this theme, with possibly a toggle button that users can choose to display or not.
I can work on this feature dev wise but I will need some guidance from the design team for choosing colors.

**Comment by joyously** [comment]
I'm all for forcing the editor to handle dark mode itself instead of making the theme have to know.
If the theme has options for colors that the user can set to anything, there's no reason why it should have to call something special just to handle that in the editor. The editor should have all of its UI specifying both foreground and background colors, so it is unaffected by the theme.
I say make the editor do it right, and all theme authors will thank you!

**Comment by senadir** [comment]
Gutenberg doesn't have control over the theme, having full site editing is still going to take a lot of time and even then Gutenberg cannot intervene there, what I wish to do is just select colors for the background & color and declare that the theme support dark mode so Gutenberg switches colors for its control components

**Comment by joyously** [comment]
Yes, GB doesn't need to control the theme. It controls its UI which shouldn't matter if the theme is dark or not.

**Comment by senadir** [comment]
it should since a dark theme will register a dark background so the control needs to adapt to that, this theme shouldn't care what Gutenberg does, it should only add support in the frontend and register dark mode when it is enabled

**Comment by joyously** [comment]
It should not matter to the theme (the theme shouldn't have to do anything different) for dark or light colors. It is the editor's responsibility to have its UI legible in every situation.

**Comment by carolinan** [comment]
This will be considered as part of https://github.com/WordPress/twentytwenty/issues/284 /#249

**Comment by ianbelanger79** [comment]
Closing because this will be completed in #284
